velox/expression/FunctionSignature.h

```diff
@@ -499,6 +499,10 @@ struct hash<facebook::velox::exec::TypeSignature> {
     for (const auto& parameter : key.parameters()) {
       val = val * 31 + this->operator()(parameter);
     }
+    if (key.rowFieldName().has_value()) {
```

Comment:
Is this a fix for a pre-existing bug? If so, let's extract this into a separate PR.